web-client/src/views/StartCase/ContactSecondaryUpdated.tsx

```diff
@@ -59,9 +67,19 @@ export const ContactSecondaryUpdated = connect<
         name: string;
         phone: string;
         paperPetitionEmail: string;
+        preferredLanguage: string;
+        preferredCommunicationMethod: string;
       };
     },
   }) {
+    const preferredLanguageLabel = isPetitioner
+      ? "If spouse has difficulty communicating in English, please state spouse's preferred language:"
+      : "If petitioner spouse has difficulty communicating in English, please state the petitioner spouse's preferred language:";
```

Comment:
Just to be sure, is this right? I would imagine it'd be something like "If the petitioner's spouse..."

Reply:
Evelyn checked with Chris Holly and this is the official text that we're going with, intentionally. Legalese is odd.